src/cs_transmog.cpp

```diff
@@ -242,32 +282,34 @@ class transmog_commandscript : public CommandScript
             return true;
         }
 
-        bool isNotConsoleAndIsPlayerOnline = handler->GetSession() && target->GetSession();
+        int locale = handler->GetSessionDbcLocale();
+        std::string setName = set->name[locale];
+        std::string nameLink = handler->playerLink(target->GetName());
```

Comment:
From phone comment for myself:
`player->GetName()` instead of `target->GetName()` or crash
Same in other parts of the code

Reply:
fixed in bfc4d32c0494e1e78361bb0813be3755ecbe163a